project_1/part_2.py

```diff
@@ -0,0 +1,152 @@
+import numpy as np
+import cv2
+import pdb
+from matplotlib import pyplot as plt
+import pdb
+
+class project:
```

Comment:
Class needs an explanation similar to how we defined the functions with 
'''
What the class is for
'''